portfolio/src/main/webapp/gallery.html

```diff
@@ -8,51 +8,33 @@
   <body>
     <div id="content">
       <h1>Gallery of Favorites</h1>
-      <table>
-        <tr class="gallery-row">
-          <td>
+      <div class="wrapper">
+        <figure class="gallery-figure">
+          <p><img src="images/blue.png" alt="9 Shades of Blue" class="gallery-image"></p>
```

Comment:
Would it be possible to remove the \<p\> tag wrapped around the \<img\> tag? 

Here and below.

Reply:
Yeah, I found that the only way I could manage to center an image in a figure was to put it in a paragraph block and center the paragraph block, which seems a little odd. Do you know of any simpler ways to do that?

Reply:
I figured it out. Done.